.perlimports.toml

```diff
@@ -0,0 +1,17 @@
+# Run `perlimports -i Slim` to tidy `use` and `require` statements.
+# https://metacpan.org/dist/App-perlimports/view/script/perlimports
+
+cache                           = false
+ignore_modules                  = []
+ignore_modules_filename         = ""
+ignore_modules_pattern          = "^Tie::"
+ignore_modules_pattern_filename = ""
+libs                            = ["lib", "CPAN", "."]
```

Comment:
If I leave that line in, I'm getting an odd error:

```
Can't locate object method "_get_callbacks" via package "Log::Dispatch::Screen" at lib/Log/Dispatch/Output.pm line 66.
```

I guess you are on Linux? I guess there's a problem with the stock Perl on Mac, which is not compatible with what it can find in those folders. `perlimports` might try to load that module from our own sub-folders, but then fail to find binaries compatible with macOS' Perl.

Reply:
And yes, as you mentioned: we should apply this module by module, as we modify them anyway. Automatic "fixing" seems to break a few things.

Reply:
I am indeed on Linux.  I think a certain number of errors are probably to be expected: I myself see, e.g., `Can't locate Win32.pm in @INC (you may need to install the Win32 module)` for some files, but assume that would be okay for someone on Windows.